docs/3.tutorials/fts/0-predeployed.md

```diff
@@ -0,0 +1,116 @@
+---
+id: predeployed-contract
+title: Pre-deployed Contract
+sidebar_label: Pre-deployed Contract
+---
+
+> Learn how to easily receive fungible tokens without doing any software development by using a readily-available FT smart contract.
+
+## Prerequisites
+
+To complete this tutorial successfully, you'll need:
+
+- [A NEAR account](/concepts/basics/accounts/creating-accounts)
+- [NEAR-CLI](/tools/near-cli#setup)
```

Comment:
same as previous "A NEAR account" shall probably change to "Near Testnet account" leading to wallet.testnet.near.org